Compile warning stack too large

```
make[1]: Entering directory '/usr/src/linux-headers-5.4.0-42-generic'
  CC [M]  /home/rountree/repos/msr-safe/msr-smp.o
/home/rountree/repos/msr-safe/msr-smp.c: In function ‘msr_safe_batch’:
/home/rountree/repos/msr-safe/msr-smp.c:108:1: warning: the frame size of 1032 bytes is larger than 1024 bytes [-Wframe-larger-than=]
  108 | }
      | ^
```

The underlying problem is allocation a `struct cpuset` on the stack.  On x86 builds this can track up to 8192 cpus.  We have not yet run into an issue with this, but I'd like to fix it all the same.

--- a/msr-smp.c
+++ b/msr-smp.c
@@ -90,7 +90,7 @@ int msr_safe_batch(struct msr_batch_array *oa)
     cpumask_var_t cpus_to_run_on;
     struct msr_batch_op *op;
 
-    zalloc_cpumask_var( &cpus_to_run_on, GFP_KERNEL );
+    zalloc_cpumask_var(&cpus_to_run_on, GFP_KERNEL);
 
     for (op = oa->ops; op < oa->ops + oa->numops; ++op)
     {

--- a/msr-smp.c
+++ b/msr-smp.c
@@ -83,6 +83,37 @@ static void __msr_safe_batch(void *info)
     }
 }
 
+#ifdef CONFIG_CPUMASK_OFFSTACK
+
+int msr_safe_batch(struct msr_batch_array *oa)
+{
+    cpumask_var_t cpus_to_run_on;
+    struct msr_batch_op *op;
+
+    zalloc_cpumask_var( &cpus_to_run_on, GFP_KERNEL );
+
+    for (op = oa->ops; op < oa->ops + oa->numops; ++op)
+    {
+        cpumask_set_cpu(op->cpu, cpus_to_run_on);
+    }
+
+    on_each_cpu_mask(cpus_to_run_on, __msr_safe_batch, oa, 1);
+
+    free_cpumask_var(cpus_to_run_on);
+
+    for (op = oa->ops; op < oa->ops + oa->numops; ++op)
+    {
+        if (op->err)
+        {
+            return op->err;
+        }
+    }
+
+    return 0;
+}
+
+#else
+
 int msr_safe_batch(struct msr_batch_array *oa)
 {
     struct cpumask cpus_to_run_on;
@@ -106,3 +137,5 @@ int msr_safe_batch(struct msr_batch_array *oa)
 
     return 0;
 }
+
+#endif //CONFIG_CPUMASK_OFFSTACK

